src/Console/FileParser.php

```diff
@@ -96,7 +96,7 @@ public function parseTemplate($template, $config)
             if ($value === ' ') {
                 $value = '';
             }
-            $this->replaceInFile($token, $value, $template);
```

Comment:
One more follow up question: Is it necessary to preserve this secret "enter a a space to blank the value" logic? If we want a value to default to the empty string, can't we use a token default value like this: `{{TOKEN:}}`?